app/views/directives/osc-routing.html

```diff
@@ -160,19 +160,26 @@ <h3>Alternate Services</h3>
           </div>
         </div>
       </div>
-
+      
       <div ng-repeat="alternate in route.alternateServices" class="form-group">
         <osc-routing-service model="alternate"
```

Comment:
nit: formatting consistency I might push `model="alternate"` down to line up with the rest of the attribs as well.